put_get_test.go

```diff
@@ -321,13 +321,18 @@ func TestPutOverwrite(t *testing.T) {
 			t.Fatalf("expected UPLOADED, got %v", s6)
 		}
 
+		rows = dbt.mustQuery("ls @~/test_put_overwrite")
+		defer rows.Close()
+		if rows.Next() {
```

Comment:
What if no rows are returned? `uploadTime` is incorrectly updated (with previous `s3` value), and test fails with strange to debug message. Maybe it would be better to assert on `rows.Next`?